Directory.Build.targets

```diff
@@ -34,4 +44,26 @@
     <PackageReference Include="Microsoft.DotNet.XUnitExtensions" />
   </ItemGroup>
 
+  <ItemGroup Condition="'$(IsTestProject)' != 'true' and '$(ExcludeFromAssemblyMetadata)' != 'true'" >
+    <AssemblyAttribute Include="System.Reflection.AssemblyDefaultAliasAttribute">
+      <_Parameter1>$(AssemblyName)</_Parameter1>
+    </AssemblyAttribute>
+    <AssemblyAttribute Include="System.Reflection.AssemblyDescriptionAttribute">
+      <_Parameter1>$(AssemblyName)</_Parameter1>
+    </AssemblyAttribute>
+    <AssemblyAttribute Include="System.Runtime.InteropServices.DefaultDllImportSearchPathsAttribute">
+      <_Parameter1>System.Runtime.InteropServices.DllImportSearchPath.AssemblyDirectory | System.Runtime.InteropServices.DllImportSearchPath.System32</_Parameter1>    
+      <_Parameter1_IsLiteral>true</_Parameter1_IsLiteral>    
+    </AssemblyAttribute>  
+    <AssemblyMetadata Include="PreferInbox">
+      <Value>True</Value>
+    </AssemblyMetadata>
+    <AssemblyMetadata Include="Serviceable">
+      <Value>True</Value>
+    </AssemblyMetadata>
+    <AssemblyMetadata Include=".NETFrameworkAssembly">
+      <Value></Value>
+    </AssemblyMetadata>
+  </ItemGroup>
```

Comment:
I think you are missing this: https://github.com/dotnet/runtime/blob/14640763bd394854d41d84b898312ec5a46aadb2/eng/versioning.targets#L29-L31

Reply:
Interesting to see that the xml attribute value is not set to a full name of the CLSCompliantAttribute type. I should probably use the full name, like the other two I already have, right?

Reply:
The good thing about adding this is that it told me which assemblies had CLSCompliant explicitly set in the source code directly: System.Json and System.Numerics.Vectors. I'll clear those attributes and will only have it set in this file.